subsys/bluetooth/host/hci_core.c

```diff
@@ -789,13 +807,22 @@ static void le_enh_conn_complete(struct bt_hci_evt_le_enh_conn_complete *evt)
 		 *
 		 * Depending on error code address might not be valid anyway.
 		 */
-		conn = bt_conn_lookup_state_le(NULL, BT_CONN_CONNECT);
+		conn = find_valid_conn(NULL);
+
 		if (!conn) {
```

Comment:
Minor coding style thing: No empty line between assignment to a variable and testing the variable's value.